### ISSUE TITLE
Surface forcing file paths contain are setup for payu users

### ISSUE THREAD

**Issue by ashjbarnes** [COSIMA/regional-mom6#93]
In the ERA5 example, the paths to each folder begin with `INPUT` as though the user were using payu. Better to just incorporate the `inputdir` that users provide when setting up the experiment.

_Originally posted by @ashjbarnes in https://github.com/COSIMA/regional-mom6/issues/79#issuecomment-1907414149_

**Comment by navidcy** [comment]
yeap, let's do that _or_ have payu as dependency of the package and explain what payu is briefly in the docs?

**Comment by navidcy** [comment]
@ashjbarnes, is this still the case?
I noticed that in the ERA5 example, `use_payu` is set to `False` (by default)

**Comment by ashjbarnes** [comment]
Yeah the `use_payu` is only turned on for the ACCESS example. To do this we'll just need a bit of logic to leave things as `INPUT/` when payu is on, and `inputdir` when it's off
